samza-core/src/main/scala/org/apache/samza/checkpoint/OffsetManager.scala

```diff
@@ -20,20 +20,19 @@
 package org.apache.samza.checkpoint
 
 
+import java.util
```

Comment:
It's needed for the hashmap. Not sure why intellij decided to import the whole package.
Made it more specific.